frontend/src/views/Studies/Milestones.js

```diff
@@ -62,11 +61,32 @@ export const Milestones = ({ sites, sitesCount, enrollmentGoal }) => {
     setSiteActivationPercentages(thresholdDates)
   }
 
+  const setPatientEnrollmentThresholds = () => {
+    let count = 1
+    const enrollment = [...enrollmentData]
+      .filter(d => !!d.actualEnrollment)
+      .sort((a, b) => a.date < b.date ? -1 : 1)
+      .map(d => ({ date: d.date, enrollment: +d.actualEnrollment }))
+
+    const thresholdDates = enrollment.reduce((dates, d) => {
+      const count = dates.indexOf('N/A')
+      let newDates = [...dates]
+      if (d.enrollment / enrollmentGoal >= (count + 1) * 0.25) {
+        newDates[count] = formatDate(new Date(d.date))
+      }
+      return [...newDates]
+    }, ['N/A', 'N/A', 'N/A', 'N/A'])
+
+    setPatientEnrollmentPercentages(thresholdDates)
+    setPatientEnrollmentPercentages(thresholdDates)
```

Comment:
@mbwatson Line 80 and 81 are duplicate. Looks like one duplicate line can be removed?